modules/rpc/README.md

```diff
@@ -0,0 +1,42 @@
+# RPC Module
```

Comment:
are you saying that there is a way to get there readmes to work with go-doc or that we should replace these reamdes with godoc?